src/views/players/PlayerView/components/AchievementCard.vue

```diff
@@ -1,70 +1,91 @@
 <template>
   <div>
     <div class="center">
-      <img :src="image" :alt="title" />
+      <img v-if="isAchieved()" :src="image" :alt="title" />
+      <img v-if="!isAchieved()" class="gray" :src="image" :alt="title" />
     </div>
     <p class="caption">{{ title }}</p>
   </div>
 </template>
 
 <script lang="ts">
+  import { ProfileAchievement } from '@/types/common-types';
+  import { values } from 'lodash';
   import { Component, Vue, Mixins, Prop } from 'vue-property-decorator';
 
   @Component({
     components: {},
   })
   export default class AchievementCard extends Vue {
-    @Prop({ required: true }) readonly image!: String;
+    @Prop({ required: true }) readonly image!: string;
 
-    @Prop({ required: true }) readonly title!: String;
+    @Prop({ required: true }) readonly title!: string;
+
+    @Prop({ required: true }) readonly achievements!: {
+      [name: string]: ProfileAchievement;
+    };
+
+    isAchieved(): boolean {
+      let b = false;
+      Object.values(this.achievements).forEach( (value) => {
+        if (value.title === this.title) {
+          b = true;
+        }
+      });
+      return b;
+    }
```

Comment:
This feels like the wrong place for this calculation. The AchievementCard ought to be a display component, and render in grayscale based on an input prop like `completed`. If `completed` is false, the card displays in grayscale. We ought to move this calculation to the ParentView component. I have an example gist of how it could work [here](https://gist.github.com/tkaragianes/dc49a42cefe3503374d5cb21618d7161).